zng/docs/spec.md

```diff
@@ -860,7 +864,7 @@ This scheme allows complex types to be embedded in other complex types, e.g., a
 #26:record[city:string,lat:LL,long:LL]
 26:[NYC;[N;40.7128;][W;74.0060;]]
 ```
-An unset value indicates a field of a `record` that wasn't set by the encoder:
+An null value indicates a field of a `record` that wasn't set by the encoder:
```

Comment:
```suggestion
A null value indicates a field of a `record` that wasn't set by the encoder:
```

Reply:
Ok, adding more on null value and type null.  Getting rid of "nil".  We did that bikeshed :-)